templates/terraform/examples/backend_service_traffic_director.tf.erb

```diff
@@ -0,0 +1,32 @@
+resource "google_compute_backend_service" "<%= ctx[:primary_resource_id] %>" {
+  provider = "google-beta"
+
+  name          = "<%= ctx[:vars]['backend_service_name'] %>"
+  health_checks = ["${google_compute_health_check.health_check.self_link}"]
+  load_balancing_scheme = "INTERNAL_SELF_MANAGED"
+  locality_lb_policy = "RING_HASH"
+  circuit_breakers {
+    max_connections = 10
+  }
+  consistent_hash {
```

Comment:
Can you use a session affinity of `HEADER_FIELD` here so you can set `http_header_name`?